test/e2e/utils/engine.ts

```diff
@@ -59,15 +60,3 @@ export const getEngineBackendWallet = async (engine: Engine) => {
   console.log("Using existing wallet", res.result[0].address);
   return getAddress(res.result[0].address);
 };
-
-export const setWalletBalance = async (
```

Comment:
thoughts on being able to explicitly set the balance on an account vs depending on the default anvil accounts to have balance?

Reply:
Again didn't need it so rather keep things minimal. Can always add more on top